OpenRA.Mods.Common/Traits/World/MusicPlaylist.cs

```diff
@@ -105,6 +102,7 @@ public MusicPlaylist(World world, MusicPlaylistInfo info)
 
 		void IWorldLoaded.WorldLoaded(World world, WorldRenderer wr)
 		{
+			Game.Sound.DisableWorldSounds = info.DisableWorldSounds;
```

Comment:
IMO it would be wise to have a short comment here explaining that this is needed to work around possible bogus existing Game.Sound state.

Reply:
Updated. Hope the wording is ok.